test/converter/excelHandlerSpec.js

```diff
@@ -189,10 +280,9 @@ describe('excelHandler', () => {
       expect(dmnContent.rules[0].outputEntries.length).to.eql(expectedRules[0].outputEntries.length);
       expect(dmnContent.rules[0].outputEntries[0].text).to.eql(expectedRules[0].outputEntries[0].text);
     });
-
   });
 
-
+  
```

Comment:
CI is failing because of a linting error. You can fix it by using `npm run lint -- --fix`.